### PR TITLE
KAA-1467: Administration Guide -> Tenant and applications management

### PR DESCRIPTION
**Pull request by Pyshankov**

### CODE REVIEW

File: doc/Administration-guide/Tenants-and-applications-management/index.md [hunk 1]
```diff
@@ -12,69 +12,22 @@ sort_idx: 40
 - [First registration](#first-registration)
 - [Kaa user types](#kaa-user-types)
 - [Kaa admin](#kaa-admin)
-  - [Configuring general settings](#configuring-general-settings)
-  - [Configuring outgoing mail settings](#configuring-outgoing-mail-settings)
-  - [Managing System CTL](#managing-system-ctl)
+  - [Managing Tenants](#managing-tenants)
```
Comment (akokhanovskyi) [inline doc/Administration-guide/Tenants-and-applications-management/index.md]: Why don't you use the standard TOC generation tag?

File: doc/Administration-guide/Tenants-and-applications-management/index.md [hunk 2]
```diff
@@ -12,69 +12,22 @@ sort_idx: 40
...
-    - [Record form](#record-form)
-  - [Working with schema form](#working-with-schema-form)
-  - [Working with record form](#working-with-record-form)
-  - [Avro UI forms GWT Sandbox](#avro-ui-forms-gwt-sandbox)
-  - [Avro schema parameters](#avro-schema-parameters)
 
-This guide explains how to manage develop applications in Kaa Admin UI and how to work with Avro UI forms.
 
-This guide also explains how to use _Avro UI forms GWT Sandbox_, which is a testing environment for Avro UI forms.
+This guide explains how to manage develop applications in Kaa Admin UI.
```
Comment (akokhanovskyi) [inline doc/Administration-guide/Tenants-and-applications-management/index.md]: What? This guide is supposed to be about managing tenants and applications.

File: doc/Administration-guide/Tenants-and-applications-management/index.md [hunk 3]
```diff
@@ -167,36 +92,6 @@ To create a new application, do the following:
...
-**NOTE:** More than one schema can be added to an ECF.
-**NOTE:** If uploaded from a file, a schema(s) should be written in the [Avro](http://avro.apache.org/docs/current/spec.html) format and describe how event classes should be grouped depending on subject areas.
-
-   <img src="attach/image2015-3-2%2014-54-7.png" width="500" height="400">
-
-A unique version number is assigned to a schema after its creation and then the schema appears as a clickable line in the **Schemas** table. To review the ECF schema details, click the appropriate schema line in the **Schemas** table. Each schema automatically splits into event classes. A full qualifier name, schema and type are shown for each event class in the table with the same name.
-
-<img src="attach/image2015-3-4 17-8-26.png" width="500" height="400">
-
 ## Managing Tenant CTL
```
Comment (akokhanovskyi) [inline doc/Administration-guide/Tenants-and-applications-management/index.md]: CTL is not in scope of this page.

**Review by akokhanovskyi — CHANGES_REQUESTED** [review]

**Comment by rasendubi** [comment]
@Pyshankov this needs update.

**Comment by rasendubi** [comment]
@akokhanovskyi please, re-review

File: doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md [hunk 4]
```diff
@@ -209,6 +209,6 @@ To provision your owner verifier, do the following:
 6. Restart kaa-node service:
 <br/>
 ``` $ sudo service kaa-node restart```
-7. Use [Admin UI]({{root_url}}Administration-guide/Tenants-and-applications-management/#adding-user-verifiers) or [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier) to create/update/delete your owner verifier instances.
+7. Use Admin UI or [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier) to create/update/delete your owner verifier instances.
```
Comment (akokhanovskyi) [inline doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md]: Administration UI

File: doc/Getting-started/index.md [hunk 5]
```diff
@@ -194,11 +194,19 @@ To access Avro UI, click the **Avro UI** button on the Sandbox main page.
...
-#### Outgoing email settings
+#### Configuring general settings
 
-Kaa uses an outgoing email server to send notifications to the users.
-There are no default settings provided for the server.
-To configure the server settings, follow the instructions in [Configuring the outgoing email server settings]({{root_url}}Administration-guide/Tenants-and-applications-management#configuring-outgoing-mail-settings).
+The **General settings** window allows you to configure the application title and application base URL parameters. These parameters are used for sending email notifications to registered users.
+To customize the general settings, click **Settings => General** settings and fill in the fields as required.
+
+<img src="attach/image2015-5-28%2017-22-54.png" width="700" height="350">
```
Comment (akokhanovskyi) [inline doc/Getting-started/index.md]: Wrong png

File: doc/Getting-started/index.md [hunk 6]
```diff
@@ -194,11 +194,19 @@ To access Avro UI, click the **Avro UI** button on the Sandbox main page.
...
+
+#### Configuring outgoing mail settings
+
+Outgoing mail settings are used to send emails to newly created users with the information about their passwords, as well as for sending other notifications.
+To customize the outgoing mail settings, click **Settings => Outgoing mail settings** and fill in the fields according to your SMTP mail server configuration.
+
+<img src="attach/image2015-5-28%2017-26-57.png" width="600" height="400">
```
Comment (akokhanovskyi) [inline doc/Getting-started/index.md]: Wrong png

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 7]
```diff
@@ -185,7 +185,7 @@ This way we get object files `.o` and shared object files `.so` that are require
         
     Copy Botan object files `${SDKTARGETSYSROOT}/usr/lib/libbotan*` to `/usr/lib` directory of Edison board.
 
-1. [Generate C++ SDK]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk).
+1. Download C++ SDK.
 1. Compile C++ SDK.
```
Comment (akokhanovskyi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:189]: 2.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 8]
```diff
@@ -185,7 +185,7 @@ This way we get object files `.o` and shared object files `.so` that are require
         
     Copy Botan object files `${SDKTARGETSYSROOT}/usr/lib/libbotan*` to `/usr/lib` directory of Edison board.
 
-1. [Generate C++ SDK]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk).
+1. Download C++ SDK.
```
Comment (akokhanovskyi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: ...from your Kaa instance.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md [hunk 9]
```diff
@@ -110,7 +110,7 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
         cp -r ${INSTALLDIR}/include/botan-1.11/botan/ ${INSTALLDIR}/include
         rm -r ${INSTALLDIR}/include/botan-1.11
 
-7. [Generate C++ SDK]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk).
+7. Download C++ SDK.
```
Comment (akokhanovskyi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md]: ...from your Kaa instance.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 10]
```diff
@@ -115,7 +115,7 @@ For more information, please refer to [the official CC3200 Getting Started Guide
 ## Creating applications based on C SDK
 
 Before creating applications based on the C SDK, you should obtain the C SDK and build a static library from it.
-To do so, generate the [C SDK in Admin UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk), then extract the archive.
+To do so, generate the C SDK in Admin UI, then extract the archive.
```
Comment (akokhanovskyi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: Administration UI

**Review by akokhanovskyi — CHANGES_REQUESTED** [review]

**Review by akokhanovskyi — APPROVED** [review]

**Pull request merged**